src/components/dialogs/reapply.vue

```diff
@@ -1,25 +1,37 @@
 <template>
-  <v-dialog max-width="40%" :value="isVisible" persistent>
+  <v-dialog max-width="35rem" :value="isVisible" persistent>
```

Comment:
Does this shrink OK if the user has a narrow viewport?

Reply:
yes it does (I tested it by narrowing my window - do you have a different way of testing this?)

Reply:
That's how I test it!